test/CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ macro(onedpl_construct_exec test_source_file _test_name switch_off_checked_itera
     target_compile_definitions(${_test_name} PRIVATE _PSTL_TEST_SUCCESSFUL_KEYWORD=1)
 
     if (NOT ${custom_define} STREQUAL "")
-        target_compile_definitions(${_test_name} PRIVATE custom_define)
+        target_compile_definitions(${_test_name} PRIVATE ${custom_define})
```

Comment:
May we describe this change as separate  issue and fix it in the independent PR ?

Reply:
Discussed offline and agreed that the CMake bugfix can be left as a part of this PR.